### ISSUE TITLE
Easy way to test theme changes

### ISSUE THREAD

**Issue by raboof** [akka/akka-paradox#29]
Currently we don't have an obvious way to test changes to the template - might be nice to have some kind of (perhaps 'sbt scripted') test project to easily test changes to the theme

**Comment by johanandren** [comment]
Yah, for the record the current way would be to 

* `sbt publishLocal` a version
* look at what snapshot-version you get
* put that version in for example akka in the `project/plugins.sbt` entry for the akka-paradox plugin 
* generate docs with `akka-docs/paradox` 
* go to `akka-docs/target/paradox/site/main`
* in this case, opening the file won't work because it wants to script-load other resources and that cannot be done from a `file://` URL, so give HTTP access locally through for example `python -m SimpleHTTPServer 8080`

**Comment by raboof** [comment]
Is your akka checkout up-to-date? This would happen if Akka was still using sbt 0.13, but we've updated Akka to sbt 1.x a while ago.

**Comment by rasummer** [comment]
@johanandren I'm trying to get the theme working locally again. 

When I generate Akka docs locally with the default `addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.18")` it works.

However if I reference a published local snapshot `addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.20+1-63a6b06a")` and then try to generate the Akka docs, it throws the following error: 

```
[error] java.lang.IllegalStateException: Expected 1 ToC roots (based on paradoxExpectedNumberOfRoots) but found 5: [index.html, camel.html, fault-tolerance-sample.html, guide/modules.html, index-futures.html]
[error] 	at com.lightbend.paradox.ParadoxProcessor.process(ParadoxProcessor.scala:82)
[error] 	at com.lightbend.paradox.sbt.ParadoxPlugin$.$anonfun$baseParadoxSettings$39(ParadoxPlugin.scala:171)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:40)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:269)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:278)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:269)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (akka-docs / Compile / paradoxMarkdownToHtml) java.lang.IllegalStateException: Expected 1 ToC roots (based on paradoxExpectedNumberOfRoots) but found 5: [index.html, camel.html, fault-tolerance-sample.html, guide/modules.html, index-futures.html]
```

Any guidance on how to fix this, or if I'm doing it incorrectly would be appreciated.

thanks Rich

**Comment by ennru** [comment]
You hit a newly introduced Paradox feature which makes sure no dangling pages exist.
I just merged https://github.com/akka/akka/pull/27457 and this should go away...

**Comment by ennru** [comment]
@rasummer It might improve your turn-around to use a smaller project such as [Akka Management](https://github.com/akka/akka-management) to try things out.

**Comment by ennru** [comment]
Fixed with #64 https://github.com/akka/akka-paradox/pull/64#issuecomment-604133511
